reference/system-databases/cluster-config.md

```diff
@@ -0,0 +1,53 @@
+---
+title: CLUSTER_CONFIG
+category: reference
+---
```

Comment:
@TomShawn Would you please help add a `summary` line for all newly added files?

Reply:
OK